src/Filesystem.cc

```diff
@@ -210,7 +210,7 @@ void DirIter::close_handle()
 
 #else  // Windows
 
-static const char preferred_separator = '\\';
+constexpr char kSdfFileSystemPreferredSeparator[] = '\\';
```

Comment:
This should be reverted. The separator is actually a single `\` character. Two backslashes are necessary to escape it.

Reply:
My bad, that was a horrific misunderstanding of the code, https://github.com/gazebosim/sdformat/pull/1264/commits/e05d4ecc1d70ed67ca1798e4eeb9882066eef645